go.mod

```diff
@@ -305,15 +305,15 @@ require (
 	gitlab.com/yawning/obfs4.git v0.0.0-20220204003609-77af0cba934d // indirect
 	go.etcd.io/bbolt v1.3.10 // indirect
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0 // indirect
-	go.opentelemetry.io/otel v1.19.0 // indirect
+	go.opentelemetry.io/otel v1.28.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlpmetric v0.42.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.42.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.19.0 // indirect
-	go.opentelemetry.io/otel/metric v1.19.0 // indirect
-	go.opentelemetry.io/otel/sdk v1.19.0 // indirect
+	go.opentelemetry.io/otel/metric v1.28.0 // indirect
+	go.opentelemetry.io/otel/sdk v1.28.0 // indirect
 	go.opentelemetry.io/otel/sdk/metric v1.19.0 // indirect
-	go.opentelemetry.io/otel/trace v1.19.0 // indirect
+	go.opentelemetry.io/otel/trace v1.28.0 // indirect
```

Comment:
Can we keep using the same go.opentelemetry.io/otel version as flashlight? v1.19.0

Reply:
@atavism  Using Otel V19, I am getting below error

```
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/provider.go:90:30: cannot use &TracerProvider{} (value of type *TracerProvider) as "go.opentelemetry.io/otel/trace".TracerProvider value in variable declaration: *TracerProvider does not implement "go.opentelemetry.io/otel/trace".TracerProvider (missing method tracerProvider)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/provider.go:170:10: cannot use t (variable of type *tracer) as "go.opentelemetry.io/otel/trace".Tracer value in return statement: *tracer does not implement "go.opentelemetry.io/otel/trace".Tracer (missing method tracer)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/span.go:161:23: cannot use (*recordingSpan)(nil) (value of type *recordingSpan) as ReadWriteSpan value in variable declaration: *recordingSpan does not implement ReadWriteSpan (missing method AddLink)
                have addLink("go.opentelemetry.io/otel/trace".Link)
                want AddLink("go.opentelemetry.io/otel/trace".Link)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/span.go:692:9: cannot use s.tracer.provider (variable of type *TracerProvider) as "go.opentelemetry.io/otel/trace".TracerProvider value in return statement: *TracerProvider does not implement "go.opentelemetry.io/otel/trace".TracerProvider (missing method tracerProvider)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/span.go:780:20: cannot use nonRecordingSpan{} (value of type nonRecordingSpan) as "go.opentelemetry.io/otel/trace".Span value in variable declaration: nonRecordingSpan does not implement "go.opentelemetry.io/otel/trace".Span (missing method AddLink)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/span.go:811:74: cannot use s.tracer.provider (variable of type *TracerProvider) as "go.opentelemetry.io/otel/trace".TracerProvider value in return statement: *TracerProvider does not implement "go.opentelemetry.io/otel/trace".TracerProvider (missing method tracerProvider)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/tracer.go:30:22: cannot use &tracer{} (value of type *tracer) as "go.opentelemetry.io/otel/trace".Tracer value in variable declaration: *tracer does not implement "go.opentelemetry.io/otel/trace".Tracer (missing method tracer)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/tracer.go:47:21: impossible type assertion: p.(*recordingSpan)
        *recordingSpan does not implement "go.opentelemetry.io/otel/trace".Span (missing method AddLink)
                have addLink("go.opentelemetry.io/otel/trace".Link)
                want AddLink("go.opentelemetry.io/otel/trace".Link)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/tracer.go:117:10: cannot use tr.newNonRecordingSpan(sc) (value of type nonRecordingSpan) as "go.opentelemetry.io/otel/trace".Span value in return statement: nonRecordingSpan does not implement "go.opentelemetry.io/otel/trace".Span (missing method AddLink)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/tracer.go:119:9: cannot use tr.newRecordingSpan(psc, sc, name, samplingResult, config) (value of type *recordingSpan) as "go.opentelemetry.io/otel/trace".Span value in return statement: *recordingSpan does not implement "go.opentelemetry.io/otel/trace".Span (missing method AddLink)
                have addLink("go.opentelemetry.io/otel/trace".Link)
                want AddLink("go.opentelemetry.io/otel/trace".Link)
/Users/jigarfumakiya/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.19.0/trace/tracer.go:119:9: too many errors

```

Reply:
That's after doing go mod download go.opentelemetry.io/otel@v1.19.0?